CHANGELOG.md

```diff
@@ -4,6 +4,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.0] - 2024-04-15
+
+### Known Issues:
+
+- Default Kconfig values may not propagate correctly if the Golioth SDK
+  is not listed first in the West manifest. To workaround this, ensure
+  that Golioth is listed first in your application's manifest.
+
+### Added:
+
+- Measure code coverage in continuous integration tests
+- Add socket hello_nrf91_offload Zephyr example to demonstrate DTLS
+  sockets offload.
+- Zephyr: add rak5010_nrf52840 support to all examples
+- ESP-IDF: add fw_update example to demonstrate OTA firmware update
+
+### Changed:
```

Comment:
We should also include the change to allow setting runtime credentials in the examples without a reboot.